app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum([player.get_rating() for player in players])
```

Comment:
The use of list comprehension with square brackets in `sum([player.get_rating() for player in players])` is unnecessary and creates a redundant list. According to the checklist, you should pass the comprehension without square brackets: `sum(player.get_rating() for player in players)` .